Feature/damage indicators and health showing

Guidoge3 opened the pull request:
damage indicators for when you get hit or when your base is under attack

Guidoge3 committed:
small push of my work so i wont lose my progress if something goes wrong

Guidoge3 committed:
damage notifier is done for archer now knight is left

Guidoge3 committed:
prefab created of ping canvas

DaanKikkert commented on PVB Linx Groep 6/Assets/Code/Scripts/Health/Healthbar_p.cs:
Rename to "HealthbarPlayer"

DaanKikkert commented on PVB Linx Groep 6/Assets/Code/Scripts/UI/C_DamageNotifier1.cs:
Rename to "CastleDamageNotifier"

DaanKikkert commented on PVB Linx Groep 6/Assets/Code/Scripts/UI/P_DamageNotifier.cs:
Rename to "PlayerDamageNotifier"

DaanKikkert commented on PVB Linx Groep 6/Assets/Code/Scripts/UI/P_DamageNotifier.cs:
```diff
@@ -0,0 +1,35 @@
+    public void ShowDamageNotifier()
+    {
+        StartCoroutine(DamageNotifierTimer());
+        Debug.Log("ur mum");
```
Remove debug.log

DaanKikkert commented on PVB Linx Groep 6/Assets/Code/Scripts/UI/C_DamageNotifier1.cs:
```diff
@@ -0,0 +1,35 @@
+    public void ShowDamageNotifier()
+    {
+        StartCoroutine(DamageNotifierTimer());
+        Debug.Log("ur mum");
```
remove debug.log

DaanKikkert commented on PVB Linx Groep 6/Assets/Code/Scripts/UI/C_DamageNotifier1.cs:
```diff
@@ -0,0 +1,35 @@
+public class C_DamageNotifier : MonoBehaviour
+{
+    [SerializeField] private Text c_DamageText;
+    [SerializeField] private float delayBetweenNotifier_C;
```
Rename both variables to no longer contain single letters and underscores, these are only for clarifying access level

DaanKikkert commented on PVB Linx Groep 6/Assets/Code/Scripts/UI/P_DamageNotifier.cs:
```diff
@@ -0,0 +1,35 @@
+public class P_DamageNotifier : MonoBehaviour
+{
+    [SerializeField] private Text p_DamageText;
+    [SerializeField] private float delayBetweenNotifier_P;
```
Rename both variables to no longer contain single letters and underscores, these are only for clarifying access level

DaanKikkert reviewed (changes requested):
Check comments for requested changes

Guidoge3 committed:
hotfix of changes needed for merge

all text containing wrong casings have been changed, script names have been changed to fix the code conventions
